src/components/ContainerBox/ContainerBox.css

```
@@ -0,0 +1,78 @@
+    justify-content: center;
+    padding: 0;
+    border: 0;
+    background: transparent;
+    cursor: pointer;
+}
+
+.container-box-icon-button svg {
+    width: 16px;
+    height: 16px;
+    fill: none;
+    stroke: currentColor;
```

Stylelint (17.10.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)